tests/core/test_views_swagger.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+import mock
+import yaml
```

Comment:
Do we have to change this to `ruamel.yaml` now that we've updated the dependency? Sorry if this is a stupid question, I've never used `ruamel.yaml` before...